client/components/domains/registrant-extra-info/ca-form.tsx

```diff
@@ -104,34 +111,27 @@ export class RegistrantExtraInfoCaForm extends PureComponent {
 			},
 		};
 
-		this.props.updateContactDetailsCache( payload );
 		this.props.onContactDetailsChange?.( payload );
 	}
 
-	handleChangeEvent = ( event ) => {
+	handleChangeEvent = ( event: {
+		target: { id: string; value: string; checked: boolean; type: string };
+	} ) => {
 		const { value, checked, type, id } = event.target;
-		const newContactDetails = {};
+		const newContactDetails: DomainContactDetails = {};
 
 		newContactDetails.extra = {
 			ca: { [ camelCase( id ) ]: type === 'checkbox' ? checked : value },
 		};
 
-		this.props.updateContactDetailsCache( { ...newContactDetails } );
-
-		if ( this.props.isManaged ) {
```

Comment:
All uses of the ccTLD forms were using `isManaged` so it has been made the default.